src/telemetry.ts

```diff
@@ -9,9 +9,7 @@ const osName = require('os-name');
 
 export const areAllTelemetryConfigsEnabled = () => {
   // respect both the overall and Stripe-specific telemetry configs
-  const enableTelemetry = vscode.workspace
-    .getConfiguration('telemetry')
-    .get('enableTelemetry', false);
```

Comment:
should we also remove this from package.json?

Reply:
This doesn't exist in our package.json because it's the VSCode-wide telemetry setting. We will still reference `stripe.telemetry.enabled` which allows users to configure Stripe telemetry separately.